tests/print_benchmarks.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
```

Comment:
Not sure I understand the purpose of this script, doesn't pytest-benchmark already print the results when run?

Also, maybe move this to `scripts/` dir

Reply:
Please see my comment on the thread in `tox.ini`!

That's a great idea, will move to `scripts/`!